aspnetcore/tutorials/build-your-first-blazor-app.md

```diff
@@ -22,7 +22,7 @@ Follow the guidance in the <xref:blazor/get-started> article to create a Blazor
 
 1. Browse to each of the app's three pages in the *Pages* folder: Home, Counter, and Fetch data. These pages are implemented by the Razor component files *Index.razor*, *Counter.razor*, and *FetchData.razor*.
 
-1. On the Counter page, select the **Click me** button to increment the counter without a page refresh. Incrementing a counter in a webpage normally requires writing JavaScript, but Blazor provides a better approach using C#.
+1. On the Counter page, select the **Click me** button to increment the counter without a page refresh. Incrementing a counter in a webpage normally requires writing code in two languages, JavaScript and C#. Blazor provides an approach that only requires one language, C#.
```

Comment:
How about this?

```suggestion
1. On the Counter page, select the **Click me** button to increment the counter without a page refresh. Incrementing a counter in a webpage would normally require writing some JavaScript, but with Blazor you can instead write C#.
```

Reply:
No future tense tho.
"Some" is very conversational and not specific.

Stand-by ... I'll make minor edits to the suggestion for another look.

Reply:
@danroth27 Try that updated suggestion :point_up:

Reply:
Looks good to me!